string id
string type
float64 positionX
float64 positionY
float64 positionZ
float64 orientationRoll
float64 orientationPitch
float64 orientationYaw
---
bool answer